### DISCUSSION TITLE
LetsEncrypt / Acme Challenge using DNS-01 instead of HTTP-01

### DISCUSSION THREAD

**Discussion by jcipolat** [drakkan/sftpgo#1432] [General]
I'd prefer to not have port 80 open on my FTP server. LetsEncrypt does not provide subnets used for validation per https://letsencrypt.org/docs/faq/#what-ip-addresses-does-let-s-encrypt-use-to-validate-my-web-server

They do support DNS-01 which seems like a less exposed validation method. Does or will sftpgo support this method?

**Comment by Elio-FairlyMade** [comment]
+1 for DNS-01 support

**Comment by pedroponte** [comment]
+1

**Comment by daFlo** [comment]
+1
